socialhome/streams/app/tests/components/AuthorBar.tests.js

```diff
@@ -0,0 +1,81 @@
+import "chai/register-should"
+import Vue from "vue"
+import BootstrapVue from "bootstrap-vue"
+import AuthorBar from "streams/app/components/AuthorBar.vue"
+
+
+Vue.use(BootstrapVue)
+
+describe("AuthorBar", () => {
+
+    describe("computed", () => {
+        describe("isUserRemote", () => {
+            it("should be the opposite of property isUserLocal", () => {
+                let propsData = {
+                    handle: "", name: "", guid: "", currentBrowsingProfileId: "", homeUrl: "", absoluteUrl: "",
+                    imageUrlSmall: "", isUserAuthor: true, isUserLocal: true, isUserFollowingAuthor: true,
+                    isUserAuthentificated: true,
+                }
+
+                let target = new AuthorBar({propsData})
+                target.isUserRemote.should.be.false
+                propsData.isUserLocal = false
+                target = new AuthorBar({propsData})
+                target.isUserRemote.should.be.true
+            })
+        })
+
+        describe("canFollow", () => {
+            it("sould be true if user is local, authentificated and not the author", () => {
```

Comment:
So yeah, I think this (and the code) should be `sould be true if user is authentificated and not the author` 👍